bl-kernel/admin/views/content.php

```diff
@@ -279,15 +280,16 @@ function table($type) {
 			'type': 'hidden',
 			'name': 'type',
 			'value': 'delete'
-		}))));
+		}).append(jQuery('<input>', {
+			'type': 'hidden',
+			'name': 'hash',
+			'value': hash
+		})))));
 
 		form.hide().appendTo("body").submit();
 	});
-});
-</script>
 
-<script>
 	// Open the tab defined in the URL
-	const anchor = window.location.hash;
-	$(`a[href="${anchor}"]`).tab('show');
-</script>
\ No newline at end of file
+	$('a[href="'+ $(location).attr('hash').slice(0,-1) +'"]').click();
```

Comment:
Why not use the previus code ? is more readable and works with bootstrap function `show()`.

Reply:
Because this also triggers the `onclick=`, so the right tab hash is set again.

Reply:
(Also, it used ``` which is very new and could cause compatibility issues)